HalloWing_Jump_Scare_Trap/hallowing_jump_scare_trap.py

```diff
@@ -0,0 +1,148 @@
+# HalloWing Jump Scare Trap
+# use PIR sensor, speaker, and servo
+import time
+import array
+import math
+import board
+import displayio
+import pulseio
+from adafruit_motor import servo
+import digitalio
+import touchio
+import audioio
+# Setup LED and PIR pins
+LED_PIN = board.D13  # Pin number for the board's built in LED.
+PIR_PIN = board.SENSE   # Pin port connected to PIR sensor output wire.
+# Setup digital input for PIR sensor:
+pir = digitalio.DigitalInOut(PIR_PIN)
+pir.direction = digitalio.Direction.INPUT
+# Setup digital output for LED:
+led = digitalio.DigitalInOut(LED_PIN)
+led.direction = digitalio.Direction.OUTPUT
+# Setup servo
+# servo = pulseio.PWMOut(board.D4, frequency=50)
+pwm = pulseio.PWMOut(board.D4, duty_cycle=2 ** 15, frequency=50)
```

Comment:
i dont think you need to do the dutycycle/frequency part, try removing those?

Reply:
I'm not sure how to define the servo object on the next line without the pwm line.

Reply:
Oh, got it, this works as you suggested now.